contracts/interfaces/IERC1363.sol

```diff
@@ -7,13 +7,14 @@ import {IERC20} from "./IERC20.sol";
 import {IERC165} from "./IERC165.sol";
 
 /**
- * @dev Interface of an ERC1363 compliant contract, as defined in the
- * https://eips.ethereum.org/EIPS/eip-1363[EIP].
+ * @title IERC1363
+ * @dev Interface of the ERC1363 standard as defined in the
+ * https://eips.ethereum.org/EIPS/eip-1363[EIP-1363].
  *
- * Defines a interface for ERC20 tokens that supports executing recipient
- * code after `transfer` or `transferFrom`, or spender code after `approve`.
+ * Defines an extension interface for ERC20 tokens that supports executing code on a recipient contract
+ * after `transfer` or `transferFrom`, or code on a spender contract after `approve`, in a single transaction.
```

Comment:
The formulation you propose could be interpreted as 

> If I call {ERC20-transfer} or {ERC20-transferFrom}, then code can be executed.

That would be incorrect, since the ERC20 functions are not modified, and don't trigger any callback. Also it doesn't say anything about where the code is executed (the important part of 1363 is that we call the receiver, we don't really care about the code that is executed in the contract)

Reply:
What about something like
```suggestion
 * Defines an extension interface for {ERC20} tokens which provides alternative versions of {ERC20-transfer}, {ERC20-transferFrom} and {ERC20-approve} that include callbacks. The callbacks allow the recipient of a transfer (or of an approval) to process the received tokens (or the approval) in the same transaction.
```